### ISSUE TITLE
Training Generative Adversarial Networks with Limited Data 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#87]
# TL;DR
Data augmentation (and the use of cycle consistency between augmented data in D) can be used to reduce overfitting. However, if the data augmentation application probability p is too large, the generated data will be affected by the augmentation. Therefore, they proposed a GAN to prevent over-fitting while preventing the influence on the generation by adaptively adjusting p. They succeeded in making a high quality image with only a few thousand data. 
![image](https://user-images.githubusercontent.com/24524018/92561062-70dd3d00-f2ae-11ea-80d1-5eaae45f2a4b.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2006.06676

# Submission Dates(yyyy/mm/dd)

# Authors and institutions

# Methods

# Results

# Comments
